### ISSUE TITLE
Bug: android.os.TransactionTooLargeException: data parcel size 790684 bytes

### ISSUE THREAD

**Issue by haiduc2511** [Yalantis/uCrop#933]
this problem occurs on both my phones when i try to edit an image that is too.. tall?
(790684 bytes is not the image's size but rather something in UCrop, i compressed my image to 1KB and this still happens)

This happens while UCrop activity's opening

This is the stack trace

FATAL EXCEPTION: main
Process: com.example.memestorage, PID: 1357
java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 790684 bytes
at android.app.servertransaction.PendingTransactionActions$StopInfo.run(PendingTransactionActions.java:160)
at android.os.Handler.handleCallback(Handler.java:907)
at android.os.Handler.dispatchMessage(Handler.java:105)
at android.os.Looper.loop(Looper.java:216)
at android.app.ActivityThread.main(ActivityThread.java:7625)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
Caused by: android.os.TransactionTooLargeException: data parcel size 790684 bytes
at android.os.BinderProxy.transactNative(Native Method)
at android.os.BinderProxy.transact(Binder.java:1149)
at android.app.IActivityManager$Stub$Proxy.activityStopped(IActivityManager.java:4005)
at android.app.servertransaction.PendingTransactionActions$StopInfo.run(PendingTransactionActions.java:144)
at android.os.Handler.handleCallback(Handler.java:907) 
at android.os.Handler.dispatchMessage(Handler.java:105) 
at android.os.Looper.loop(Looper.java:216) 
at android.app.ActivityThread.main(ActivityThread.java:7625) 
at java.lang.reflect.Method.invoke(Native Method) 
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524) 
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)

**Comment by haiduc2511** [comment]
Figured it out (after 3 hours of my precious life), using UCrop in fragment will cause this problem (since you'll parse the image from the activity to the fragment?)

TLDR: use activity instead
